Core/Security/Locations.py

```diff
@@ -55,8 +57,15 @@ def getCAsLocation():
   # /etc/grid-security/certificates
   casPath = "/etc/grid-security/certificates"
   if os.path.isdir(casPath):
-    return casPath
-  # No CA's location found
+    if len(os.listdir(casPath)) != 0:
+      lastChange = max(os.stat(file).st_mtime for file, _, _ in os.walk(casPath))
+      lastChange = datetime.datetime.strptime(time.ctime(lastChange), "%a %b %d %H:%M:%S %Y")
+      elapsedTime = datetime.datetime.now() - lastChange
+      if elapsedTime.days < 2:
+        return casPath
+      else:
+        gLogger.warn("X509_CERT_DIR points to a corrupt certificates directory")
```

Comment:
Who is the target for this warning? Is it users? Because `gLogger.warn` doesn't get printed by default in scripts, still, right?
Also what does corrupt mean? Does it mean potentially outdated? Or is the else for the empty directory one indent level to the left?

Reply:
I honestly don't know what good behaviour to put here. This is kind of addressing https://github.com/DIRACGrid/DIRAC/issues/5083

Reply:
`gLogger.warn` indeed is not printed by default in scripts (which is maybe wrong, but not for today). What's wrong with using `gLogger.error`?

Reply:
I am concerned about this rather arbitrary check that I put `if elapsedTime.days < 2:` is this is sensible? So I was more inclined to WARN then ERROR.